test/soca/runtime_vars.sh

```diff
@@ -5,18 +5,16 @@ project_binary_dir=$1
 project_source_dir=$2
 
 # Prepare runtime environement
-export CDATE=2018041512        # Center of current cycle date
+export PDY=20180415        # Center of current cycle date
+export cyc=12        # Center of current cycle date
 export GDATE=2018041506        # Center of previous cycle date
 export gcyc=$(echo $GDATE | cut -c9-10)
 export CDUMP=gdas
 export GDUMP=gdas
 export OPREFIX=${CDUMP}.t${cyc}z.
-export PDY=20180415
-export cyc=12
-export OPREFIX=${CDUMP}.t${cyc}z.
 export assim_freq=6            # DA window
 export COMOUT=${project_binary_dir}/test/soca/3dvar
-export DATA=${project_binary_dir}/test/soca/3dvar/ocnanal_${CDATE}
+export DATA=${project_binary_dir}/test/soca/3dvar/ocnanal_${PDY}${cyc}
```

Comment:
See my comment in g-w about this path

Reply:
We have an issue to address the extra dir level, if that's what you were reffering to @CoryMartin-NOAA , I removed PDY from the name if that's what your comment was about.